thirdparty/CMakeLists.txt

```diff
@@ -24,8 +24,8 @@ if(BUILD_TOKENIZERS)
         # Prebuilt dependencies
         if(WIN32)
             set(extra_libs "${CMAKE_BINARY_DIR}/_deps/fast_tokenizer-src/lib/core_tokenizers.dll"
```

Comment:
we need to drop code related to `core_tokenizers` as we have removed them in OpenVINO Tokenizers

can be done as a separate PR as it seems current one will be green soon and finally be merged.

Reply:
https://github.com/openvinotoolkit/openvino.genai/pull/1710